src/Compilers/CSharp/Test/Semantic/Semantics/PrimaryConstructorTests.cs

```diff
@@ -10261,7 +10245,7 @@ struct S1(Color Color)
 {
     public S1() : this(
         Color.M1(
-            new Color()))
```

Comment:
>new Color() [](http://example.com/codeflow?start=12&length=11)

Should we test that the ambiguity error is raised even when one method is inapplicable due to argument type mismatch, e.g., calling `Color.M1(new Color())` in this test?

Reply:
> Should we test that the ambiguity error is raised even when one method is inapplicable due to argument type mismatch, e.g., calling Color.M1(new Color()) in this test?

I think inapplicability scenarios are adequately covered by other targeted tests. For example, ```ParameterCapturing_027_ColorColor_MemberAccess_InstanceInapplicableAndStaticApplicableDueToArguments_Method```. Also look for other tests with "Inapplicable" in the name. There is also ```ParameterCapturing_063_ColorColor_MemberAccess_InstanceAndStaticAmbiguity_Method_ApplicabilityDueToArgumentsNotConsidered```.